tools/lsp/preview.rs

```diff
@@ -161,6 +162,18 @@ fn search_for_parent_element(root: &ElementRc, child: &ElementRc) -> Option<Elem
     None
 }
 
+// triggered from the UI, running in UI thread
```

Comment:
(for a later refactoring: keep all the code that is running in the UI thread in a separate module so we don't need these comments)